lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -71,10 +72,23 @@ def set_mpiexec_global_options(self):
             self.mpiexec_global_options['pml'] = 'ucx'
             # disable uct btl, see http://openucx.github.io/ucx/running.html
             self.mpiexec_global_options['btl'] = '^uct'
+            if self.options.debuglvl > 3:
+                os.environ['UCX_LOG_LEVEL'] = 'debug'
+                self.mpiexec_global_options['pml_ucx_verbose'] = self.options.debuglvl
+
+            if self.options.stats:
+                # report UCX stats at the end to stdout
+                os.environ['UCX_STATS_TRIGGER'] = 'exit'
+                os.environ['UCX_STATS_DEST'] = 'stdout'
         else:
             # default PML (ob1) uses Byte Transport Layer (BTL)
             self.mpiexec_global_options['btl'] = self.device
 
+        if self.options.debuglvl > 3:
+            value = [None, "--%(name)s"]
```

Comment:
Please add a comment here to explain that the `None` here is not a mistake, this looks really weird without knowing that this is going to be processed. And you should really use a tuple `(None, "--%(name)s")` here imho, a list is mutable which makes no sense at all here.